build.gradle

```diff
@@ -37,6 +39,17 @@ subprojects {
 
   kotlin.jvmToolchain(17)
```

Comment:
I think we can remove this now, too? No reason to use such an ancient JDK. Developers should always be using the latest one.

Reply:
Do you mean it should use the user JDK (and the JDK defined in the GitHub Actions workflows) or that I should upgrade it to 21?

Reply:
The former, the "user"/global JDK. It'll fail if someone tries to use older than 11, but otherwise our actions should be using 23.

Reply:
Removed and version upgraded to 23 on the workflows 👍